### ISSUE TITLE
Issue about model training on N-MNIST

### ISSUE THREAD

**Issue by TJXTT** [chan8972/Enabling_Spikebased_Backpropagation#4]
Hi,

I'm trying to implement LeNet using the N-MNIST dataset but the accuracy does not increase as the training step growing.  I load the N-MINST dataset by PySNN ( https://github.com/BasBuller/PySNN).

Can you please tell me how to train the model on N-MNIST?

Your help will be appreciated.
Thanks

**Comment by chan8972** [comment]
We are not aware of PySNN code. We have preprocessed the N-MNIST dataset (similar to "https://github.com/chan8972/Spike-FlowNet/blob/master/encoding/split_coding.py") and use dataloader to feed the data through the network. Since N-MNIST contains sparse event data, there typically requires a higher degree of variance for the weight parameter initialization.
